confgenerator/confgenerator.go

```diff
@@ -97,6 +99,152 @@ func (m *Metrics) generateOtelPipelines() (map[string]otel.Pipeline, error) {
 	return out, nil
 }
 
+func (uc *UnifiedConfig) GenerateFluentBitConfigsModular(logsDir string, stateDir string, hostInfo *host.InfoStat) (string, string, error) {
```

Comment:
Just wondering, would it be better to rename the old `GenerateFluentbitConfigs` to `GenerateFluentbitConfigsLegacy`, and name this one `GenerateFluentbitConfigs`?..

Reply:
I'm going to delete the legacy one in a further commit on this branch and update the name of this function at the same time.